Mighty-Menorahs/reactive-nutshell
# Ep delete

# Description

Please look at the delete button and make sure it deletes the task from the DOM and from JSON. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout ep-delete
Please look at the delete button and make sure it deletes the task from the DOM and from JSON. 
Please make sure the "Add New Task" feature works as well. 




# Checklist:
